### ISSUE TITLE
Why did you use dual license?

### ISSUE THREAD

**Issue by Smithangshu** [adeptoas/sweet-modal#26]
Hi,

Thank you for developing and distributing such a wonderful JQuery plugin. I have a question, I am developing a web application for my client who will hold the copyright of the application when development is completed. He will use this application for commercial business purpose. 

So can we as developer, will be able to use this plugin in my client's commercial web application without any legal issue from your side?

Best regards,
Smith

**Comment by bluefirex** [comment]
Yes, that is correct. You can choose either license, which allows you to also use it in commercial applications. Please bear in mind that the jQuery version of SweetModal is longer actively maintained by me and relies on pull requests.
